index.html

```diff
@@ -180,25 +180,38 @@ <h2 class="hed"> Skills</h2>
         </section>
         <section>
             <div class="finishing">
-                <p class="last-r">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
-                <form action="/action_page.php" id="acts">
-                    <input type="text" placeholder="Full name" id="nam">
-                    <input type="email" placeholder="email address" id="nam">
-                    <textarea name="enter-t" id="enter-t" cols="30" rows="5">Enter text here</textarea>
+                <div class="last-r">
+                    <p class="last-r1">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+                </div>
+                
+                <form action="https://formspree.io/f/xzbqzrje" id="acts" method="post">
+                    <fieldset class="inputs">
+                        <div class="contn">
+                            <input type="text" placeholder="Full name" id="nam" maxlength="30">
+                        </div>
+                       <div class="contn">
+                        <input type="email" placeholder="Email address" id="nam" required>
+                       </div>
+                        
+                        <textarea name="enter-t" id="enter-t" cols="30" rows="5" class="txt-area" maxlength="500">Enter text here</textarea>
+                    </fieldset>
+                       
+
                     <button type="submit" class="in-touch">Get in touch</button>
                 </form>
-               
+    
+            </div>
+            <div class="footer">
                 <ul class="social">
-                    <a href="https://github.com/ikoote1"><img src="./images/Vector1.png" alt="github"></a>
-                    <a href="https://www.linkedin.com/public-profile/settings?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_self_edit_contact-info%3Bmblb4PfaTiGNkcYIq9q9%2BA%3D%3D"><img src="./images/Vector2.png" alt="Linkedln"></a>
-                    <a href="https://freewil.com/ikoote/"><img src="./images/Vector3.png" alt="freewill"></a>
-                    <a href="https://twitter.com/ikootepreim/"><img src="./images/Vector4.png" alt="twitter"></a>
-                    <a href="https://messenger.com/preim"><img src="./images/Vector5.png" alt="messenger"></a>
+                    <a href="https://github.com/ikoote1"><img src="./images/Vectorr1.png" alt="github"></a>
+                    <a href="https://www.linkedin.com/public-profile/settings?lipi=urn%3Ali%3Apage%3Ad_flagship3_profile_self_edit_contact-info%3Bmblb4PfaTiGNkcYIq9q9%2BA%3D%3D"><img src="./images/Vectorr2.png" alt="Linkedln"></a>
+                    <a href="https://freewil.com/ikoote/"><img src="./images/Vectorr3.png" alt="freewill"></a>
```

Comment:
- [ Required] Please consider fixing this image URL, it's not displaying on the page because it's name is not correct. Please see the markup below on how you can fix this.

- [ Required ] Please consider adding the top divider or `border-top` on the social media section as shown in the Figma template screenshot below.
```HTML
  <a href="#"><img src="./images/vectorr2.png" alt="Linkedln"></a>
```
## Figma template
![Screenshot from 2023-01-18 00-32-48](https://user-images.githubusercontent.com/16392046/213017279-7f7c9737-d857-496e-b66f-ec2615f9bb51.png)


## Your design
![Screenshot from 2023-01-18 00-30-36](https://user-images.githubusercontent.com/16392046/213016609-f8ae9d32-7afc-40d6-b127-abc13f316df9.png)